src/components/icons/arrowDown.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react"
```

Comment:
Hmmm do we not build components from automatically here yet?

Reply:
no, not yet

Reply:
Can you create a follow up for this? This is something that we use in other projects, makes sense to align the approach

Reply:
Created a task for it: https://autoricardo.atlassian.net/browse/CAR-5906